### ISSUE TITLE
Configuration for `max_staleness` should be string with explicit format, instead of integer

### ISSUE THREAD

**Issue by slothkong** [fishtown-analytics/docs.getdbt.com#5218]
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/blog/announcing-materialized-views

### What part(s) of the page would you like to see updated?

While trying out materialized views on BigQuery, I seem to have found an error in the code snippet. The funny thing is there does not seem to be an "Edit this page" button on the [actual docs website](https://docs.getdbt.com/blog/announcing-materialized-views).

Bottom line is, `max_staleness = 60` returned an error, but `max_staleness = 'INTERVAL 60 MINUTE` went through.

### Additional information

I tested with:
- `dbt-core`  v1.7.3
- `dbt-bigquery`  v1.7.1
